CONTRIBUTING.md

```diff
@@ -34,25 +34,25 @@ Se recommend using VS Code with eslint extensions, which will automatically chec
     -   [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository) so that you can make your changes without affecting the original project until you're ready to merge them.
 
-3.   Install or update to **Node.js v16**. 
+3.   Install or update to **Node.js v16**.
 
 3.   Create a working branch ```feature/my-cool-feature``` or ```bugfix/issue-14``` and start with your changes!
 
 ### Commit your update
 
-Please run ```npm run build``` before committing, to update the dist-files, and to make sure every test and check passes. If you make changes to any function Interface, or to JSDoc in general, you should also run ```npm run build:docs``` to update the generated documentation.
+Please run ```npm run build``` before committing, to update the dist-files, and to make sure that every test and check passes. If you make changes to any function Interface, or to JSDoc in general, you should also run ```npm run build:docs``` to update the generated documentation.
 
 See [package.json](/package.json) scripts section for all available scripts.
 
-Then, commit the changes once you are happy with them. 
+Then, commit the changes once you are happy with them.
 
 ### Pull Request
 
-When you're finished with the changes, create a pull request, also known as a PR.
+When you're done with the changes, create a pull request, also known as a PR.
 -   Don't forget to [link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if you are solving one.
--   Enable the checkbox to [allow maintainer edits](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so the branch can be updated for a merge.
+-   Enable the checkbox to [allow maintainer edits](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so that the branch can be updated for a merge.
 Once you submit your PR, a team member will review your proposal. We may ask questions or request for additional information.
--   We may ask for changes to be made before a PR can be merged, either using [suggested changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) or pull request comments. You can apply suggested changes directly through the UI. You can make any other changes in your fork, then commit them to your branch.
+-   We may ask for changes to be made before a PR can be merged, either using [suggested changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) or pull request comments. You can apply suggested changes directly through the UI. You can make any other changes in your fork, then commit them to your branch, the changes will automatically be reflected in the PR.
```

Comment:
You can make any other changes in your fork, then commit them to your branch. The changes will automatically be reflected in the PR.